### PR TITLE
add sustainability page 

### PR DESCRIPTION
**Pull request by hjohns28**

### CODE REVIEW

**Comment by FabriceCastel** [comment]
Note: work branched off of https://github.com/hjohns28/step/pull/7

File: portfolio/src/main/webapp/style.css [hunk 1]
```diff
@@ -33,6 +33,48 @@
...
+    left: 350px;
+    top: 300px;
+    opacity: 0;
+}
+
+.plastic {
+    position: absolute;
+    left: 600px;
+    top: 300px;
+    opacity: 0px;
```
Comment (kristen-law) [inline portfolio/src/main/webapp/style.css]: Don't need px here

**Review by kristen-law — CHANGES_REQUESTED** [review]
LGTM, just one small comment

File: portfolio/src/main/webapp/style.css (hunk 1 above)
Comment (hjohns28) [reply portfolio/src/main/webapp/style.css]: Thank you for the comment! I changed that later on, it should be in a later pull request. Should I still change it here?

Comment (hjohns28) [reply portfolio/src/main/webapp/style.css]: Sorry, you can ignore my above comment - I changed it!

**Review by FabriceCastel — APPROVED** [review]
LGTM

**Review by kristen-law — APPROVED** [review]

**Pull request merged**